src/sentry/static/sentry/app/components/version.tsx

```diff
@@ -49,25 +54,39 @@ type Props = {
 
 const Version = ({
   version,
-  orgId,
+  organization,
   anchor = true,
   preserveGlobalSelection,
   tooltipRawVersion,
   withPackage,
   projectId,
-  className,
   truncate,
-}: Props) => {
+  className,
+  location,
+}: ReactRouter.WithRouterProps & Props) => {
   const LinkComponent = preserveGlobalSelection ? GlobalSelectionLink : Link;
   const versionToDisplay = formatVersion(version, withPackage);
 
+  let releaseDetailProjectId: null | undefined | string | string[];
+  if (preserveGlobalSelection) {
```

Comment:
Hm I am not sure I like the idea of having an empty condition..I'd write it in another way